roles/openshift_prometheus/templates/prometheus.rules.j2

```diff
@@ -0,0 +1,13 @@
+groups:
+- name: example-rules
+  interval: 30s # defaults to global interval
+  rules:
+  - alert: Node Down
+    expr: up{job="kubernetes-nodes"} == 0
+    annotations:
+      miqTarget: "ContainerNode"
+      severity: "HIGH"
+      message: "{{ '{{' }}{{ '$labels.instance' }}{{ '}}' }} is down"
+{% if openshift_prometheus_external_rules is defined and openshift_prometheus_external_rules is not none %}
+  - {{ openshift_prometheus_external_rules }}
+{% endif %}
```

Comment:
@simon3z I will condition the creation of the example rules with a variable. (I don't want to lose this example code, and I also want to keep the ability to test).

> Also, the openshift_prometheus_external_rules could it be a file that is just copied in an additional ConfigMap?

Yes. I can change it so the user will pass a rule file path if you think this is better